src/core/call/filter_fusion.h

```diff
@@ -117,6 +122,90 @@ class AdaptMethod<T, void (Call::*)(A, Derived*), method,
   Derived* filter_;
 };
 
+template <class T>
+struct Void {
+  typedef void type;
+};
+
+// primary template handles types that have no nested ::Call member:
+template <class, class = void>
+struct has_call_member : std::false_type {};
```

Comment:
let's call this `HasCallMember`

... I think it's also possible to write:

```
template <class, class = void>
constexpr const bool kHasCallMember = false;
template <class T>
constexpr const bool kHasCallMember<T, std::void_t<typename T::Call>> = true;
```

Reply:
Fixed